charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.10.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 3.1.5
+version: 3.2.6
```

version jump looks odd:

```suggestion
version: 3.3.0
```